storage-proofs-porep/build.rs

```diff
@@ -8,4 +8,6 @@ fn cfg_if_nightly() {
 }
 
 #[rustversion::not(nightly)]
-fn cfg_if_nightly() {}
+fn cfg_if_nightly() {
+    println!("cargo::rustc-check-cfg=cfg(nightly)");
```

Comment:
I'm not sure if it's correct. I would put print it in all cases, hence put this line into `main()` instead.